definitions/checks/foreman/check_external_db_evr_permissions.rb

```diff
@@ -0,0 +1,55 @@
+module Checks
+  module Foreman
+    class CheckExternalDbEvrPermissions < ForemanMaintain::Check
+      metadata do
+        label :external_db_evr_permissions
+        for_feature :foreman_database
+        description 'Check that external DBs have proper EVR extension permissions'
+        tags :pre_upgrade
+        confine do
+          feature(:foreman_database) && !feature(:foreman_database).local? && feature(:katello)
+        end
+      end
+
+      def run
+        # No check is needed if the evr extension does not exist
+        evr_exists = find_evr_exists
+        return if !evr_exists
```

Comment:
I think you could just turn this into more of a `evr_exists?` method, so that you can reduce these two lines to:

```
return unless evr_exists?
```